Gravity-tube/src/Pages/SingleVideoPage/SingleVideoPage.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import Navbar from '../../Components/Navigation/Navbar';
+import VideoDetailPage from './Component/VideoDetailPage';
+
+function SingleVideoPage() {
+  return (
+    <>
+     <div className="content">
+    <Navbar/>
```

Comment:
put the navbar above the routes component that way it wont re render on every page